var/spack/repos/builtin/packages/sina/package.py

```diff
@@ -0,0 +1,101 @@
+import os
+
+from spack import *
+
+
+class Sina(CachedCMakePackage):
+    """Sina C++ Library"""
+
+    homepage = 'https://github.com/LLNL/Sina'
+    url = 'https://github.com/LLNL/Sina/releases/download/v1.10.0/sina-cpp-1.10.0.tar.gz'
+
+    version('develop', git='git@github.com:LLNL/Sina.git',
+            submodules=True, branch='develop')
+    version('1.10.0', sha256='b34379ce8cc5eca5a0f16893053fac75be14c2109d1beed4c6d48e11f9b281c7')
+
+    variant('docs', default=False,
+            description='Allow generating documentation')
+    variant('adiak', default=False,
+            description='Create interface for calling Sina through Adiak')
+    # Availability of the adiak version with optional mpi varies
+    variant('adiak-no-mpi', default=False,
+            description='Create interface for calling Sina through Adiak')
```

Comment:
It looks like this juts controls a variant on a dependency, but there's no need for a separate option for that.  You can just do:

```console
$ spack install sina ^adiak~mpi
```

and then you don't need the same piece of information represented twice.

Reply:
I did not realize I could do this. I will clean this up along with changing the conduit line below.